onnxruntime/contrib_ops/webgpu/bert/bias_split_gelu.cc

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/webgpu/shader_helper.h"
+#include "core/providers/webgpu/webgpu_supported_types.h"
+#include "contrib_ops/webgpu/bert/bias_split_gelu.h"
+#include "contrib_ops/webgpu/bert/gelu.cc"
+#include "contrib_ops/webgpu/webgpu_contrib_kernels.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace webgpu {
+
+ONNX_OPERATOR_KERNEL_EX(
+    BiasSplitGelu,
+    kMSDomain,
+    1,
+    kWebGpuExecutionProvider,
+    (*KernelDefBuilder::Create())
+        .TypeConstraint("T", WebGpuSupportedFloatTypes()),
+    BiasSplitGelu);
+
+static int64_t GetMaxComponents(int64_t size) {
```

Comment:
Can this function go into utilities so that this does not have to be repeated?

Reply:
Should I create a webgpu_utils.h/cc file for this or should I add to the utils file that is for all providers?